modules/3d/misc/python/test/test_odometry.py

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env python
+
+from tabnanny import check
+import numpy as np
+import cv2 as cv
+import unittest
+
+from tests_common import NewOpenCVTests
+
+class odometry_test(NewOpenCVTests):
+    def test_OdometryDefault(self):
+        depth = self.get_sample('cv/rgbd/depth.png', cv.IMREAD_ANYDEPTH).astype(np.float32)
+        #check rotation from 0 to 4 degree
+        for degree in range(0, 40, 5):
```

Comment:
Python tests should check Python bindings only.

Not the accuracy through passing of different input values.

---

Python test should be as fast as possible.

Reply:
left only one test case for each algo